### PR TITLE
[Cordova Demo App] Push dialog doesn't display for Cordova app

### PR DESCRIPTION
**Pull request by russelarms**
* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Fix no notification dialog is shown on the main page.

## Related PRs or issues

[AB#65348](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/65348)

### CODE REVIEW

File: demoapp/www/js/push.js [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
+
+    if (pushNotification.customProperties && Object.keys(pushNotification.customProperties).length > 0) {
+        message += '\nCustom properties:\n' + JSON.stringify(pushNotification.customProperties);
+    }
+
+    alert("title = " + title + "; message = " + message)
+}
+
+function updateNotificationListener(isEnabled) {
+    //This is how you can add listener for notifications.
```
Comment (MatkovIvan) [inline demoapp/www/js/push.js]:
```suggestion

    // This is how you can add listener for notifications.
```

File: demoapp/www/js/push.js [hunk 2]
```diff
@@ -9,34 +42,15 @@ $(document).bind('pageinit', function () {
...
-        }
-
-        if (pushNotification.customProperties && Object.keys(pushNotification.customProperties).length > 0) {
-            message += '\nCustom properties:\n' + JSON.stringify(pushNotification.customProperties);
-        }
-
-        alert("title = " + title + "; message = " + message)
-    }
-
     //This is how you can check whether push is enabled.
```
Comment (MatkovIvan) [inline demoapp/www/js/push.js]:
```suggestion
     // This is how you can check whether push is enabled.
```

File: demoapp/www/js/push.js [hunk 3]
```diff
@@ -9,34 +42,15 @@ $(document).bind('pageinit', function () {
 
     var updatePush = function() {
         $("#btn_toggle_push").html(pushEnabled ? ENABLED_LBL : DISABLED_LBL);
+
         //This is how you can add listener for notifications.
```
Comment (MatkovIvan) [inline demoapp/www/js/push.js]:
```suggestion
         // This is how you can add listener for notifications.
```

**Comment by russelarms** [comment]
Initialization in 'pageinit' doesn't work, it produces:
` I/chromium: [INFO:CONSOLE(44)] "Uncaught ReferenceError: AppCenter is not defined", source: file:///android_asset/www/js/push.js (44)`
So I moved the initialization of the notification listener to `deviceready` event.

File: demoapp/www/js/push.js [hunk 4]
```diff
@@ -1,6 +1,40 @@
...
+    // This is how you can add listener for notifications.
+    if (isEnabled) {
+        AppCenter.Push.addEventListener('notificationReceived', onNotificationReceived); 
+    } else {
+        AppCenter.Push.removeEventListener('notificationReceived', onNotificationReceived); 
+    }
+}
+
+function onDeviceReady() {
+    AppCenter.Push.isEnabled(function (isEnabled) {
```
Comment (annakocheshkova) [inline demoapp/www/js/push.js]: Why are we making this check twice? Let's move the `pushEnabled` variable initialization to the place where it's first called.

File: demoapp/www/js/push.js [hunk 5]
```diff
@@ -9,35 +43,16 @@ $(document).bind('pageinit', function () {
...
     var updatePush = function() {
         $("#btn_toggle_push").html(pushEnabled ? ENABLED_LBL : DISABLED_LBL);
-        //This is how you can add listener for notifications.
-        if (pushEnabled) {
-            AppCenter.Push.addEventListener('notificationReceived', onNotificationReceived); 
-        } else {
-            AppCenter.Push.removeEventListener('notificationReceived', onNotificationReceived); 
-        }
+
+         // This is how you can add listener for notifications.
```
Comment (jwallra) [inline demoapp/www/js/push.js]: I don't think this comment is really necessary since it's already in the method

File: demoapp/www/js/push.js [hunk 6]
```diff
@@ -9,35 +43,16 @@ $(document).bind('pageinit', function () {
...
-
-        if (pushNotification.customProperties && Object.keys(pushNotification.customProperties).length > 0) {
-            message += '\nCustom properties:\n' + JSON.stringify(pushNotification.customProperties);
-        }
-
-        alert("title = " + title + "; message = " + message)
-    }
-
-    //This is how you can check whether push is enabled.
+     // This is how you can check whether push is enabled.
```
Comment (jwallra) [inline demoapp/www/js/push.js]: delete extra space
```suggestion
    // This is how you can check whether push is enabled.
```

**Pull request merged**